src/scenes/AcceleratorRouter/ParticipantProjects/ProjectProfileGisMaps.tsx

```diff
@@ -96,59 +97,132 @@ const ProjectProfileGisMaps = () => {
     }
   }, [boundariesData]);
 
+  const basePlantingMapData = useMemo(() => {
+    if (plantingSitesData) {
+      return MapService.getMapDataFromGisPlantingSites(plantingSitesData);
+    }
+    return undefined;
+  }, [plantingSitesData]);
+
+  const [processedSiteData, setProcessedSiteData] = useState<any>(null);
+
+  // Process site data asynchronously
```

Comment:
Not sure if it matters much, but do we usually put state items all at the top of a component?